docs/book/src/cronjob-tutorial/testdata/project/cmd/main.go

```diff
@@ -85,9 +93,28 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	disableHTTP2 := func(c *tls.Config) {
```

Comment:
Could we add a comment explained why it has been made?

Reply:
Addressed in https://github.com/kubernetes-sigs/kubebuilder/pull/3734/commits/2375fa34d34e520eded52b4ac0e1c63795d9a5a9